artifactory/cli.go

```diff
@@ -3340,6 +3348,9 @@ func releaseBundleDistributeCmd(c *cli.Context) error {
 	if c.NArg() != 2 {
 		return cliutils.PrintHelpAndReturnError("Wrong number of arguments.", c)
 	}
+	if c.IsSet("max-wait-minutes") && !c.IsSet("sync") {
+		return cliutils.PrintHelpAndReturnError("flag --max-wait-minutes can't be used without --sync", c)
```

Comment:
We use the term "option" rather than "flag" in all messages.
Need to be changed in the other message in this function.